assets-and-images.md

```diff
@@ -53,17 +57,30 @@ different locales or regions, reading directions, etc.
 
 ## Working with assets
 
-Your application can access its assets through the
-[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html)
-interface. The two main methods on an asset bundle allow you to load a string
+Your application can access its assets through an
+[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html).
+The two main methods on an asset bundle allow you to load a string
 (`loadString`) or an image (`loadImage`) out of the bundle, given a logical key.
 The logical key maps to the path to the asset specified in the `flutter.yaml`
-file at build time. The application's asset bundle is accessible through the
-[DefaultAssetBundle](http://docs.flutter.io/flutter/widgets/DefaultAssetBundle/of.html)
-widget's `of` method.
+file at build time.
+
+Each Flutter application has a
+[rootBundle](http://docs.flutter.io/flutter/services/rootBundle.html)
+for easy access to the main asset bundle. You can use the `rootBundle`
```

Comment:
There's nothing magical about `rootBundle`. If you call `takeRootBundle` before the `rootBundle` property is initialised, then you can use mojo to read the main asset bundle yourself.